version.mk

```diff
@@ -48,3 +48,25 @@ helm-version:
 		sed -i'.bak' -e "s_^\\.version:\ .*_.version: \\&version \"$${VERSION}\"_g" examples/chart/$${CHART}/Chart.yaml || exit 1; \
 		rm -f examples/chart/$${CHART}/Chart.yaml.bak; \
 	done
+
+# This awk is a little hacky - xml is not line-based - but this avoids adding
+# another tool (such as xmlstarlet or plutil) as a requirement of this repo.
```

Comment:
I'm happy adding a tool that understands XML in place of the code below, so we can rest easy that we are not messing up the Info.plist file if something goes wrong. Also +1 if it helps readability.

Reply:
Fair enough. I think I'll knock together a simple tool in Go. No more deps but will be more robust.

Reply:
Sounds good!